RunTest

```diff
@@ -923,13 +917,30 @@ for bmode in "$test8" "$test16" "$test32"; do
 
   if [ $doheap = yes ] ; then
     echo $titleheap
-    $sim $valgrind ./pcre2test -q $setstack $bmode $testdata/testinputheap testtry
+    $sim $valgrind $pcre2test -q $setstack $bmode $testdata/testinputheap testtry
     checkresult $? heap-$bits ""
   fi
 
 # End of loop for 8/16/32-bit tests
 done
 
+
+# ------ Special EBCDIC Test -------
+
+if [ $doebcdic = yes ]; then
```

Comment:
minor nitpick: Philip seems to like a space between "]" and ";"

Reply:
I may be wrong, but I think at one point in the historical past the space was necessary - or I got confused about it - so I got into the habit of separating [ and ] from their surroundings. (Might have been different in different shells?) Anyway, I'm not dogmatic about it.

Reply:
Done.